src/integration/pages/homePage/canonicalTests.js

```diff
@@ -37,4 +40,18 @@ export default ({ service, pageData, displayAds }) => {
       });
     }
   });
+
+  describe('VJ Embed', () => {
+    const vjEmbedComponent = document.getElementById('VJ-Embed');
+
+    if (service === 'hindi') {
```

Comment:
We don't have a homepage test running for hindi (only for kyrgyz & arabic). We have 2 options:
- add a new homepage test for hindi 
- update one of the existing homepage fixture data to have a VJ embed

Normally I would tend to say, let's not add another one and just use what we've got, BUT because we're doing the work to add the newsroom collective thing to the Hindi footer (and other indian services), I actually think we should add a new homepage test for Hindi, which means it will be much easier to add an integration test for the new footer thing once we've done that piece of work.

Can you copy the pattern from the kyrgyz/arabic homepage test and add a new one for hindi?

Reply:
added a new homepage test for hindi by copying the pattern from kyrgyz/arabic. Thanks.